### DISCUSSION TITLE
Better mobile access to sidebar in documentation

### DISCUSSION THREAD

**Discussion by rodja** [zauberzeug/nicegui#811] [Ideas / Feature Requests]
As suggested by u/koera on [Reddit](https://www.reddit.com/r/Python/comments/12u2b9w/comment/jh8uk10/?utm_source=share&utm_medium=web2x&context=3), we should make it obvious how to show/hide the sidebar in the documentation. Also make the sections collapsible.

**Comment by falkoschindler** [comment]
@rodja Have a look at https://github.com/zauberzeug/nicegui/commit/a6a881968758efcd53003610845aaea1dff189d0. Should work quite nicely on all devices.

**Reply by rodja** [reply]
Very nice. I just changed the drawer-button-icon because it was the same as the menu for navigating between the topics: 

<img width="639" alt="Screen Shot 2023-04-24 at 19 37 05" src="https://user-images.githubusercontent.com/131391/234076098-8260d730-19b0-421a-b1b5-b3266e8b62e8.png">

**Reply by falkoschindler** [reply]
I know, they have been identical.
Somehow I don't like the "menu_book". And I think I have seen other website with two identical menus.

**Reply by falkoschindler** [reply]
Alternative proposal:
![Screenshot 2023-04-25 at 10 19 29](https://user-images.githubusercontent.com/5767091/234218884-85f8a2bb-129f-4f21-9f27-f027234d97de.png)

**Reply by falkoschindler** [reply]
Ok, we agreed on https://github.com/zauberzeug/nicegui/discussions/811#discussioncomment-5716731.
